### PR TITLE
PSP-7069 : Show ACQ team (organization) in filter by ACQ team in Acqu…

### PR DESCRIPTION
**Pull request by eddherrera**
…isition List View

image

### CODE REVIEW

File: source/backend/api/Areas/Acquisition/Controllers/AcquisitionFileController.cs [hunk 1]
```diff
@@ -206,14 +206,14 @@ public IActionResult GetAcquisitionFileOwners([FromRoute] long id)
         [HasPermission(Permissions.AcquisitionFileView)]
         [HasPermission(Permissions.ContactView)]
         [Produces("application/json")]
-        [ProducesResponseType(typeof(IEnumerable<PersonModel>), 200)]
+        [ProducesResponseType(typeof(IEnumerable<AcquisitionFileTeamModel>), 200)]
         [SwaggerOperation(Tags = new[] { "acquisitionfile" })]
         public IActionResult GetAcquisitionTeamMembers()
         {
             // TODO: This endpoint needs to be fixed to support ORGANIZATIONS as team members (which were introduced by PSP-6960)
```
Comment (devinleighsmith) [inline source/backend/api/Areas/Acquisition/Controllers/AcquisitionFileController.cs:213]: this todo appears to have been addressed, can it be removed?

File: source/frontend/src/features/acquisition/list/interfaces.ts [hunk 2]
```diff
@@ -61,3 +91,20 @@ export interface MultiSelectOption {
...
+const getParameterIdFromOptions = (
+  options: MultiSelectOption[],
+  selector: IdSelector = 'P',
+): string => {
+  if (!options.length) {
+    return '';
+  }
+
+  var filterOrgItems = options.filter(option => String(option.id).startsWith(selector));
+  console.log(filterOrgItems);
```
Comment (devinleighsmith) [inline source/frontend/src/features/acquisition/list/interfaces.ts:104]: this as well.

**Review by devinleighsmith — APPROVED** [review]

**Review by devinleighsmith — APPROVED** [review]

File: source/backend/api/Areas/Acquisition/Controllers/AcquisitionFileController.cs (hunk 1 above)
Comment (eddherrera) [reply source/backend/api/Areas/Acquisition/Controllers/AcquisitionFileController.cs:213]: removed

File: source/frontend/src/features/acquisition/list/interfaces.ts [hunk 3]
```diff
@@ -36,22 +42,46 @@ export class AcquisitionFilterModel {
...
-    newModel.acquisitionTeamMembers =
-      person?.id === undefined
-        ? []
-        : [{ id: person.id.toString(), text: formatApiPersonNames(person) }];
     newModel.projectNameOrNumber = model.projectNameOrNumber;
     newModel.searchBy = model.searchBy;
     newModel.pin = model.pin;
     newModel.pid = model.pid;
     newModel.address = model.address;
+    console.log(model);
```
[the remark this answers is not in the document]
Comment (eddherrera) [reply source/frontend/src/features/acquisition/list/interfaces.ts:57]: removed

File: source/frontend/src/features/acquisition/list/interfaces.ts (hunk 2 above)
Comment (eddherrera) [reply source/frontend/src/features/acquisition/list/interfaces.ts:104]: removed

**Pull request merged**